tsconfig.base.json

```diff
@@ -3,7 +3,8 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "strict": true,
-    "moduleResolution": "node",
+    // All packages are built with ESBuild, so we can use `moduleResolution: 'bundler'`
+    "moduleResolution": "Bundler",
```

Comment:
Even though they are built with esbuild, the runtime would still break if we import a path without `.js`. esbuild doesn't append the `.js` automatically it seems. Would it be more accurate to use `node16` instead? Or maybe figure out adding the `.js`?

I think `bundler` make more sense for user's astro projects though.

Reply:
Well not blocking the PR for sure, we can also tackle this existing issue in a later PR.